setup.py

```diff
@@ -2,13 +2,16 @@
 
 setup_kwargs = dict(
     name='netsuite',
-    version='0.4.0',
+    version='0.4.1',
     description='Wrapper around Netsuite SuiteTalk Web Services and Restlets',
     packages=['netsuite'],
     include_package_data=True,
     author='Jacob Magnusson',
     author_email='m@jacobian.se',
-    url='https://github.com/jmagnusson/netsuite',
+    fork_author='Stephen Opal'
+    fork_email='sno@merit.edu'
+    fork_org='Merit Network, Inc.'
```

Comment:
I love the intent here, but this dictionary will be passed/used elsewhere and invalid kwargs aren't ideal. Can we switch these to be our own, clobbering the original author's so we don't add unexpected kwargs?

Reply:
Since we intend to push this upstream we probably shouldn't touch these for the time being.

Reply:
I'm confused.  I could not find a setup.py kwargs reserved word dictionary for this so I assumed there is not one.  I would appreciate a pointer to such a specification.

Reply:
https://packaging.python.org/tutorials/packaging-projects/